crates/re_viewer/src/ui/recordings_panel.rs

```diff
@@ -0,0 +1,104 @@
+use re_data_store::StoreDb;
+use re_log_types::Time;
+use re_viewer_context::{SystemCommand, SystemCommandSender, ViewerContext};
+use time::macros::format_description;
+
+#[cfg(not(target_arch = "wasm32"))]
+use re_ui::UICommandSender;
```

Comment:
If this is only used from within `add_button_ui` I suggest adding the `use` statement to the contents that function (reducing the scope of the binding, and making it easier to move the function)